Gem/Code/Source/LightCullingExampleComponent.cpp

```diff
@@ -96,7 +96,7 @@ namespace AtomSampleViewer
         m_sampleName = "LightCullingExampleComponent";
 
         // Add some initial lights to illuminate the scene
-        m_settings[(int)LightType::Point].m_numActive = 150;
+        m_settings[(int)LightType::Point].m_numActive = 10;
```

Comment:
This will change the golden screenshot. you will need to update that too.

Reply:
Oh Thanks. I'll update the screenshot as well.